CHANGELOG.md

```diff
@@ -5,10 +5,44 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.2.0] - 2020-07-02
+### Added
+* Overview document for [feature highlights in v0.2.0](https://github.com/Project-MONAI/MONAI/blob/master/docs/source/highlights.md)
+* Type hints and static type analysis support
+* `MONAI/research` folder
+* `monai.engine.workflow` APIs for supervised training
+* `monai.inferers` APIs for validation and inference
+* 7 new tutorials and examples
+* 3 new loss functions
+* 4 new event handlers
+* 8 new layers, blocks, and networks
+* 12 new transforms, including post-processing transforms
+* Public dataset accessing APIs and utilities
+* Dataset APIs for persistent caching, zipping, and grouping
```

Comment:
What does "grouping" mean here? Could you please also help add the `ArrayDataset`?